authority/tls.go

```diff
@@ -153,6 +102,13 @@ func (a *Authority) Sign(csr *x509.CertificateRequest, signOpts SignOptions, ext
 			http.StatusInternalServerError, errContext}
 	}
 
+	// FIXME: This should be before creating the certificate.
```

Comment:
has this been fixed?

Reply:
It's not super critical, but we should fix it.